goddard/httpd.coffee

```diff
@@ -3,11 +3,15 @@
 express			= require('express')
 bodyParser 		= require('body-parser')
 session 		= require('express-session')
+raven 			= require('raven');
 RedisStore 		= require('connect-redis')(session);
 
 # create the instance to setup and use
 app = express()
 
+# Should be the first item listed
+app.use(raven.middleware.express.requestHandler(process.env.SENTRY_DSN or ''))
```

Comment:
Perhaps only call `app.use` if the `SENTRY_DSN` is defined? Not sure if it trips up over an empty string as a DNS.